tests/test_vtttime.py

```diff
@@ -49,10 +50,17 @@ def test_hours(self):
         self.time.minutes += 42
         self.assertEqual(self.time.hours, 1)
 
-    def test_shifting(self):
+    def test_shifting_forward(self):
         self.time.shift(1, 1, 1, 1)
         self.assertEqual(self.time, (1, 1, 1, 1))
 
+    def test_shifting_backwards(self):
+        self.time.shift(-1, -1, -1, -1)
+        self.assertEqual(self.time, (-2, 58, 58, 999))
+        self.time = WebVTTTime(1,2,3,4)
+        self.time.shift(-1, -1, -1, -1)
+        self.assertEqual(self.time, (0, 1, 2, 3))
+
```

Comment:
what if the shifting ends up with subtitles with negative times? does the library raise an exception?